src/atlas/extractRoiFromAtlas.m

```diff
@@ -34,9 +34,10 @@
 
     roiName = regexprep(roiName, '(Left )|(Right )', '');
 
+    prefix = '';
     if strcmp(hemisphere, 'L')
       prefix = 'Left ';
-    elseif strcmp(hemisphere, 'L')
+    elseif strcmp(hemisphere, 'R')
```

Comment:
ahh you are finding the bugs on your own! I'll lose my title of "hunter queen of bugs"

Reply:
Well I did not have a test for that one but soon...